examples/kubernetes/connectivity-check/network.cue

```diff
@@ -7,8 +7,13 @@ package connectivity_check
 _networkCheck: {
 	metadata: labels: component: "network-check"
 }
+
+_trafficCheck: {
+    metadata: labels: traffic: "external"
```

Comment:
nit: it's possible to expand the _networkCheck: section to include this, for example:

```
_networkCheck: {
    metadata: labels: {
        component: "network-check"
        traffic: "external"
    }
}
```

Reply:
Not sure if we can merge this together, as `pod-to-a` will have traffic as external as well.